apps/omg_watcher/lib/api/inflight_exit.ex

```diff
@@ -0,0 +1,106 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.API.InflightExit do
+  @moduledoc """
+  Module provides API for transactions
+  """
+
+  alias OMG.API.State.Transaction
+  alias OMG.API.Utxo
+  alias OMG.Watcher.DB
+  alias OMG.Watcher.ExitProcessor
+
+  require Utxo
+
+  @type in_flight_exit() :: %{
+          in_flight_tx: binary(),
+          input_txs: binary(),
+          input_txs_inclusion_proofs: binary(),
+          in_flight_tx_sigs: binary()
+        }
+
+  @doc """
+  Returns arguments for plasma contract function that starts in-flight exit for a given transaction.
+  """
+  @spec get_in_flight_exit(binary) :: {:ok, in_flight_exit()} | {:error, atom}
+  def get_in_flight_exit(txbytes) do
```

Comment:
`txbytes` or `tx_bytes` - I'm mentioning this as a few lines further there is `raw_tx_bytes`